src/index.html

```diff
@@ -15,8 +15,85 @@
       rel="stylesheet"
       href="./style.css"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <div class="div">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Company Logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <a
```

Comment:
Use the semantic tag ul and li for list of links 
<img width="149" alt="image" src="https://github.com/user-attachments/assets/a91c9c74-47fb-48b5-8ca1-ad7cfa2e4f35">

Reply:
It`s done